python_modules/libraries/dagster-sigma/dagster_sigma/resource.py

```diff
@@ -245,19 +245,22 @@ def reconstruction_metadata_key(self) -> str:
 
     def build_defs(
         self,
-        context: DefinitionsLoadContext,
+        context: Optional[DefinitionsLoadContext] = None,
```

Comment:
Similar to my comment on the other PR, I think we should just take this parameter out for now. We can always add it back later.